content/_index.html

```diff
@@ -42,6 +42,35 @@ <h1 class="headline1 font-medium">
   </div>
 </section>
 
+<section class="glue-page gpb-8">
+  <a id="get-started"></a>
+  <div class="glue-grid gpt-6">
+    <div class="col-span-4 sm:col-span-10 sm:col-start-2 lg:col-span-4 lg:col-start-2">
+      <h2 class="headline2 gmb-4">
+        Join the Community
+      </h2>
+    </div>
+    <div class="col-span-4 sm:col-span-10 sm:col-start-2 lg:col-span-6">
+      <p class="gmb-2 headline5 intro-para">
+        Welcome to Transparency.dev, a community dedicated to promoting transparency across various ecosystems and pioneering its integration into new domains.
```

Comment:
nit: is it worth expanding to "application domains" given the history of CT?

Reply:
+1 good should, will fix